### DISCUSSION TITLE
importing problem

### DISCUSSION THREAD

**Discussion by sagoto80** [Cyfrin/foundry-full-course-cu#1178] [Q&A]
```
import {fundFundMe} from "../../script/InterAction.s.sol";
```

**Comment by xyz899** [comment]
Hi !

Could you give more thought to what  exactly is happening ?

If you have a problem with importing the script be sure to check the correct path is precised above. If you are for example making a new script and need to call another one in the same file you import by starting with `import {fundFundMe} from "./script/InterAction.s.sol";`

 if the file you're making is outside of the script file `use "../"`
 Be sure there aren't any typos on your file or when youre importing it

**Reply by sagoto80** [reply]
sorry for your time

**Reply by xyz899** [reply]
No problem ! Don't forget to mark this Discussion as completed

**Reply by sagoto80** [reply]
```[27943] InterAction::run()
    ├─ [3925] DevOpsTools::get_most_recent_deployment("FundMe", 31337 [3.133e4]) [delegatecall]
    │   ├─ [0] VM::readDir("./broadcast", 3) [staticcall]
    │   │   └─ ← the path broadcast is not allowed to be accessed for read operations
    │   └─ ← the path broadcast is not allowed to be accessed for read operations
    └─ ← the path broadcast is not allowed to be accessed for read operations
    ```
    i have problem with InteraAction.sol 
    is it affect importing problem?

**Reply by xyz899** [reply]
It should not affect the imports but still please check that your dependencies are up-to-date by using `foundryup` and close and reopen your project
